cypress/integration/login.js

```diff
@@ -1,3 +1,7 @@
+let currentDate = new Date();
+let day = currentDate.getDate();
+console.log(day);
```

Comment:
While console logging surely helps during development, pushing it into source control is not a very good practice, since they most of the time have no value post-development. Let's delete this line :)